.github/workflows/ci_docs.yml

```diff
@@ -115,13 +124,6 @@ jobs:
         shell: bash
         run: ./koch doc --git.commit:devel
 
```

Comment:
note: logic was moved to kochdocs (so it also applies to local docs and is a 1-liner)